### PR TITLE
Couple of py tests

### PR DESCRIPTION
**Pull request by jsarha**

### CODE REVIEW

**Comment by tomba** [comment]
Set the +x flag for the .py scripts so that they can be executed directly

File: py/tests/modeset_event.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            callback(key.fileobj, mask)
+
+def eventhandler(event):
+    print("Received %s event successfully (seq %d time %f)" %
+          (event.type, event.seq, event.time))
+
+card = pykms.Card()
+sel = selectors.DefaultSelector()
+sel.register(card.fd, selectors.EVENT_READ, readdrm)
+
```
Comment (tomba) [inline py/tests/modeset_event.py]: Not saying anything should be changed, but just pointing out that "readdrm" here is just plain data. So you don't _need_ to pass a function here, you could just have the "waitevent" function and do everything in that, without needing a separate "readdrm" function.

File: py/tests/modeset_event.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        break
+
+fb = pykms.DumbFramebuffer(card, mode.hdisplay, mode.vdisplay, format);
+pykms.draw_test_pattern(fb);
+
+# Disable request
+card.disable_planes()
+
+print("Seting %s to %s using %s" % (conn.fullname, mode.name, format))
+
```
Comment (tomba) [inline py/tests/modeset_event.py]: Typo above

File: py/pykms/pykmsbase.cpp [hunk 3]
```diff
@@ -71,6 +71,7 @@ void init_pykmsbase(py::module &m)
 
 	py::class_<Plane, Plane*>(m, "Plane",  py::base<DrmPropObject>())
 			.def("supports_crtc", &Plane::supports_crtc)
+			.def("get_formats", &Plane::get_formats)
```
Comment (tomba) [inline py/pykms/pykmsbase.cpp]: "get_foo()" should almost always be a property in python

File: py/pykms/pykmsbase.cpp [hunk 4]
```diff
@@ -138,8 +139,10 @@ void init_pykmsbase(py::module &m)
 			.value("ABGR8888", PixelFormat::ABGR8888)
 
 			.value("RGB888", PixelFormat::RGB888)
+			.value("BGR888", PixelFormat::RGB888)
```
Comment (tomba) [inline py/pykms/pykmsbase.cpp]: Wrong pixelformat

**Comment by jsarha** [comment]
Can't see anything wrong with this request, so I won't resend this.

**Comment by tomba** [comment]
Applied the patches

**Pull request closed without merging**